### DISCUSSION TITLE
Printer not laying a purge line

### DISCUSSION THREAD

**Discussion by robodruid** [bassamanator/Sovol-SV06-firmware#168] [Q&A] [answered]
The printer isn't laying down a purge line at the start of a print. The purge line macro is there, but the printer won't do it. It just moves to the front left corner, parks for a second to finish heating up, then starts the print

**Comment by bassamanator** [comment]
Not really possible if the start gcode is correct. Post it here.

**Reply by robodruid** [reply]
M104 S0 ; Stops PS/SS from sending temp waits separately
M140 S0
PRINT_START BED=[first_layer_bed_temperature] HOTEND=[first_layer_temperature[initial_extruder]]


Using the PS start gcode, but I use orcaslicer

**Reply by robodruid** [reply]
Still not laying a purge line. It goes to the front left still for a few seconds and just starts the print

**Comment by bassamanator** [comment, marked as answer]
Please see Adjust Your Slicer section.

Make this as the answer and close discussion when ready.

**Reply by robodruid** [reply]
Thak you so much. Don't know how I missed that
